### ISSUE TITLE
[A11y] DIsabled navigation link should have a noticeable color

### ISSUE THREAD

**Issue by michaelktiufb** [hlxsites/accenture-newsroom#425]
1. Navigate to https://tempnewsroom.accenture.com/news/2023/accenture-acquires-on-service-group-to-help-insurers-in-germany-improve-operational-resiliency-and-drive-business-growth
2. Check the disabled navigation link (Prev or next)

ER: The disabled link has a color of #000000
AR: The disabled link had a color of #96aee3;
